test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -291,6 +291,34 @@ def test_amount_localization
     assert_equal 2000, capture_response.params['amount']
   end
 
+  # def test_cross_border_classification_export
+  #   skip('"cross_border_classification" is only allowed to be sent for certain accounts (not the account we use for testing).')
```

Comment:
Instead of commenting out this test, you can skip it like you have here. Since the tests in Active Merchant don't use RSpec, the actual `skip` method won't work here, but you can use `omit` to achieve the same thing.